### ISSUE TITLE
make:controller --api produces "The --api option does not exist" error

### ISSUE THREAD

**Issue by Josh865** [laravel/framework#23506]
- Laravel Version: 5.5.39
- PHP Version: 7.1.9
- Database Driver & Version: MySQL 5.7.19

### Description:

When attempting to create an API resource controller using the `--api` switch, the process fails and the following error message is displayed:

`The "--api" option does not exist.`

Running the `php artisan make:controller` command _without_ the `--api` flag works as expected.

### Steps To Reproduce:

Run the following command (taken straight from the docs):

`php artisan make:controller API/PhotoController --api`

**Comment by staudenmeir** [comment]
The `--api` option was added in Laravel 5.6.

**Comment by cyrildewit** [comment]
I don't think that your Laravel version supports this feature. If you take a look at the [docs for 5.5](https://laravel.com/docs/5.5/controllers), you will not find anything about it.

**Comment by wanpixel** [comment]
so what is the solution?
